cpu/nrf52/radio/nrf802154/nrf802154.c

```diff
@@ -35,6 +36,28 @@
 #define ENABLE_DEBUG    (0)
 #include "debug.h"
 
+#define NRF802154_RF_DBM_SENSITIVITY (-100)    /**< Radio RF Sensitivity */
+#define NRF802154_ED_RSSIOFFS (-93)            /**< Radio ED offset */
+#define NRF802154_ED_RSSISCALE (5U)            /**< Radio RSSI scale */
```

Comment:
Where did you get these values from, can you point to a document? I couldn't find them in the nRF52840 product specification.

Reply:
sure! All this values are in https://infocenter.nordicsemi.com/index.jsp?topic=%2Fps_nrf52833%2Fradio.html&cp=4_1_0_5_17_11 (Clear Channel Assessment section)